curricula/templates/curricula/partials/code_studio_pull_through/level_contents.html

```diff
@@ -38,16 +38,16 @@ <h3>Student Instructions</h3>
     {% if level.type == "StandaloneVideo" and not pdf%}
     <div class="level-video">
         <iframe allowfullscreen="" frameborder="0" height="480" src="{{ level.video_youtube|no_autoplay }}" width="853"></iframe>
-        <p><a href="{{ level.video_download }}">Download Video</a></p>
+        <p><a href="{{ level.video_download }}">{% trans "Download Video" %}</a></p>
     </div>
     {% endif %}
 
     <!-- Curriculum Reference level -->
     {% if level.type == "LevelGroup" %}
     <div class="level-group">
         <p>
-            This level is an assessment or survey with multiple questions.
-            To view this level click the "View on Code Studio" link.
+            {% trans "This level is an assessment or survey with multiple questions." %}
+            {% blocktrans %}To view this level click the "View on Code Studio" link.{% endblocktrans %}
```

Comment:
Why does this one need the `blocktrans` tag?

Reply:
`blocktrans` auto-escapes characters (in this case the double quotes)